libcloud/compute/base.py

```diff
@@ -1260,8 +1260,9 @@ def delete_key_pair(self, key_pair):
         raise NotImplementedError(
             'delete_key_pair not implemented for this driver')
 
-    def wait_until_running(self, nodes, wait_period=3, timeout=600,
-                           ssh_interface='public_ips', force_ipv4=True):
+    def wait_until_running(self, nodes, ex_cloud_opts=None, wait_period=3,
```

Comment:
Can you please move this new argument to the end so it's the last argument?

This way the change is backward compatible and doesn't break existing code if using non keyword arguments.